Fw/Buffer/Buffer.fpp

```diff
@@ -16,4 +16,37 @@ module Fw {
                   $size: FwSizeType
                 ) -> Fw.Buffer
 
+  @ Interface for allocating and deallocating buffers
+  interface BufferAllocation {
```

Comment:
This is already defined in `Svc/Interfaces/BufferAllocation.fpp` - likely belong there with the BufferManager components